scrapy/czlscrape/__init__.py

```diff
@@ -1,7 +1,11 @@
 import os
+import time
+import logging
 
 if 'SENTRY_DSN' in os.environ:
     import logging
     from raven.handlers.logging import SentryHandler
     from raven.conf import setup_logging
     setup_logging(SentryHandler(os.environ['SENTRY_DSN'], level=logging.WARN))
+
+logging.Formatter.converter = time.gmtime
```

Comment:
@xlotlu @mgax : what about `datetime.utcnow()` ? That should give us the right value.
```python
>>> from datetime import datetime
>>> datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
'2017-04-06T10:14:50Z'
```

Regarding timezones, I understand what you're saying about convenience of reading the time and date in our own timezone, but this is more of a presentation concern rather than what it should be stored in.

Your suggestion to use the same timezone on all machines is good, but it should be applied with UTC as timezone. Hardware clocks should be set to UTC, because it is unambiguous. 

Also, storing datetimes in anything other than UTC means that we need to store both the value and the timezone offset.

You are right, we will need to parse the dates in the scraped data and interpret it as being in local Romanian time.